source/DontTouchMe.hx

```diff
@@ -0,0 +1,136 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxObject;
+import flixel.math.FlxPoint;
+import flixel.util.FlxColor;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+
+/**
+ * ...
+ * @author Keith Cissell
+ */
+class DontTouchMe extends Enemy 
+{
+	private var position:FlxPoint;
+
+	// Movement variables
+	private var xAccel:Float = 30;
+	private var xSlowdown:Float = 60;	
+	private var xMaxSpeed:Float = 30;
+		
+	// Hitboxes
+	public var takeDamageBox:FlxObject;
+	public var giveDamageBox:FlxObject;
+		
+	/**
+	 * Intializer
+	 *
+	 * @param	X	Starting x coordinate
+	 * @param	Y	Starting y coordinate
+	 */
+	public function new(?X:Float = 0, ?Y:Float = 0) 
+	{
+		super(X, Y, AssetPaths.DontTouchMe__png);
+				
+		name = "DontTouchMe";
+		position = getPosition();
+		
+		// Initialize gravity. Assumes the currentState has GRAVITY property.
+		acceleration.y = (cast FlxG.state).GRAVITY;
+		maxVelocity.y = acceleration.y;
+		
+		// Setup movement
+		maxVelocity.x = xMaxSpeed;
+		groundMovement( -1);
+		
+		// Set hitboxes
+		takeDamageBox = new FlxObject((X + 5), Y, 22, 5);
+		giveDamageBox = new FlxObject((X + 1), (Y + 5), 30, 27);
```

Comment:
I'd like it if you'd replace these raw numbers with some sort of variables so their meaning is clearer.